pysat/instruments/pysat_testing.py

```diff
@@ -119,6 +119,11 @@ def load(fnames, tag=None, sat_id=None, sim_multi_file_right=False,
         If True, the loaded file list time index is shifted by 5-minutes.
         This shift is actually performed by the init function.
 
+    .. deprecated:: 2.3.0
```

Comment:
In other deprecations like this, I've added the new kwarg, so they can use either method for this release.  Recommend doing that here, and in the rest of the testing instruments.